files/mods.json

```diff
@@ -546,7 +541,7 @@
         "creator": "Biscuit",
         "discordcode": "qBkzcgdPqU",
         "command": "/sba",
-        "description": "Essential QOL Skyblock mod with a plethora of indispensable (yet outdated) features, locking slots, timers, and a ton more to discover!",
+        "description": "QOL Skyblock mod with a plethora of (outdated) features, like slot locking, timers, and other stuff!",
```

Comment:
Since NEU's slot locking is arguably better and should be used over SBA's, I think slot locking here should be replaced with something like the custom warp menu or... idek.. what does SBA even have that's really that useful anymore lol.

Reply:
to be honest it’s kinda shit and I wish we could remove it